content/blog/hcl-vs-pulumi/index.md

```diff
@@ -0,0 +1,1551 @@
+---
+title: "Pulumi vs HCL: Understanding the Language Differences in Infrastructure as Code"
+allow_long_title: true
+date: 2024-07-09
+meta_desc: "Compare Pulumi and HCL for infrastructure as code. Discover how general-purpose languages boost productivity and collaboration cloud management."
+meta_image: meta.png
+authors:
+- engin-diri
+tags:
+- pulumi
+- terraform
+- infrastructure-as-code
+- cloud-management
+- hcl
+---
+
+The Java Language Architect at Oracle, Brian Goetz, author of Java Concurrency in Practice, has this to how declarative
+languages can be a double-edged sword:
+
+![brian-goetz-tweet](brian-goetz-tweet.png)
+
+## Summary
```

Comment:
you can probably remove Summary as a header since its just part of the intro

Reply:
done!